backend/de.metas.business/src/main/java-legacy/org/compiere/model/CalloutOrder.java

```diff
@@ -200,9 +191,12 @@ public String docType(final ICalloutField calloutField)
 		}
 
 		order.setInvoiceRule(getDefaultInvoiceRule());
+
+		// Invoice Rule, Payment Rule - POS Order
 		if (MOrder.DocSubType_POS.equals(docSubType))
 		{
 			order.setInvoiceRule(X_C_Order.INVOICERULE_Immediate);
+			order.setPaymentRule(PaymentRule.Cash.getCode());
```

Comment:
why set the paymentrule here?

Reply:
Decided with Torby to leave it like that for now